src/metal.cs

```diff
@@ -4164,24 +4554,609 @@ interface MTLCounterSampleBuffer {
 		NSData ResolveCounterRange (NSRange range);
 	}
 
-	[NoiOS, NoTV, Mac (10,15)]
-	[BaseType (typeof(NSObject))]
+	[iOS (14,0), TV (14,0), Mac (10,15)]
+	[BaseType (typeof (NSObject))]
 	interface MTLCounterSampleBufferDescriptor : NSCopying
 	{
-		[NoiOS, Mac (10, 15)]
+		[Mac (10, 15), NoiOS, NoTV] // fails on intro simulator and device
```

Comment:
Do we have an issue tracking this that we can link?

Reply:
It is in the ignore file and in the commit message :)